### PR TITLE
Updates to socket.io config

### PR DESCRIPTION
**Pull request by eric-silva-61**
### What this PR does (required):
- Added configuration to socket.io settings

### Screenshots / Videos (required):
- Post at least one screenshot to show the feature in action

### Any information needed to test this feature (required):
- Dashboard is currently making a socket connection back to the server using the socket context.
- Cookies are available on the "socket.handshake.headers.cookie" objecty

### Any issues with the current functionality (optional):
- was unsure how to implement user tracking, so it will be implemented when the messaging fills in

### CODE REVIEW

**Review by rajivtitus — COMMENTED** [review]
- Please make sure the socket is open only when a user is logged in
- Parse the cookie, for eg: use socketCookieParser
- Verify jwt token extracted from cookie (jwt.verify)
- Log a message to console

Please follow the above steps :)

**Comment by eric-silva-61** [comment]
Yeah thanks for the clarification.  I'm not sure what to do about your first point but I implemented the other 3

**Comment by eric-silva-61** [comment]
OK I think I get it.  I added a try/catch and disco the conn if token not verified

File: server/app.js [hunk 1]
```diff
@@ -1,70 +1,89 @@
...
-    origin: "*",
+    origin: 'http://localhost:3000',
+    credentials: true,
   },
 });
 
-io.on("connection", (socket) => {
-  console.log("connected");
+io.on('connection', (socket) => {
+  console.log('connected');
```
Comment (rajivtitus) [inline server/app.js]: We only want to display this message when the user has been verified

**Review by rajivtitus — APPROVED** [review]
Looks good for a start! Eventually we also need the ability to keep track of which users are currently connected

I've made one small comment, please have a look

**Pull request merged**